.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 
 node_js:
+  - '10'
+  - '8'
   - '6'
-  - '4'
```

Comment:
can you extract these changes to a separate PR? if we bump the required Node versions then it's a breaking change and deserves a dedicated PR so that we can label it properly in the changelog.